general-superstaq/general_superstaq/_init_vars.py

```diff
@@ -1,2 +1,12 @@
 API_URL = "https://superstaq.super.tech"
 API_VERSION = "v0.2.0"
+TOKEN_PUBLIC_KEY_E = 65537
+TOKEN_PUBLIC_KEY_N = "170085480843227966251730824714595515546847560229328345394016543390109446338"
```

Comment:
Why is it broken up like this? if it's because we need to please a linter i think this is the exception where we tell the linter to ignore this line.

Reply:
we could also use e.g.
```python
TOKEN_PUBLIC_KEY_N = int(
    "123456789123456789123456789123456789123456789123456789123456789123456789123456789"
    "123456789123456789123456789123456789123456789123456789123456789123456789123456789"
    "123456789123456789123456789123456789123456789123456789123456789123456789123456789"
    ....
)
so that it's both a single command and actually an `int`